neqo-transport/src/events.rs

```diff
@@ -99,23 +114,104 @@ impl ConnectionEvents {
     }
 
     pub fn connection_state_change(&self, state: State) {
+        // If closing, existing events no longer relevant.
```

Comment:
A question: do we guarantee that all streams are read when we switch to Closing state? Can there be some data sitting in stream buffer?
In case of Firefox this will be true, because Firefox closes the connection only when there are no more streams.

Reply:
Right now we drop all stream data when the connection enters `Closing`. If this is incorrect behavior please open an issue.

Reply:
This is fine, I just want to double check the behavior, that we drop all stream data.